src/CoinbaseSmartWalletFactory.sol

```diff
@@ -5,22 +5,36 @@ import {LibClone} from "solady/utils/LibClone.sol";
 import {CoinbaseSmartWallet} from "./CoinbaseSmartWallet.sol";
 
 /// @title Coinbase Smart Wallet Factory
+///
 /// @notice CoinbaseSmartWallet factory, based on Solady's ERC4337Factory.
+///
 /// @author Coinbase (https://github.com/coinbase/smart-wallet)
 /// @author Solady (https://github.com/vectorized/solady/blob/main/src/accounts/ERC4337Factory.sol)
 contract CoinbaseSmartWalletFactory {
-    /// @dev Address of the ERC4337 implementation.
+    /// @notice Address of the ERC-4337 implementation used to deploy new cloned accounts.
     address public immutable implementation;
 
+    /// @notice Reverted when trying to create a new `CoinbaseSmartWallet` account without any owner.
     error OwnerRequired();
 
+    /// @notice Factory constructor used to initialize the implementation address to use for future
+    ///         ERC-4337 account deployments.
+    ///
+    /// @param erc4337 The address of the ERC-4337 implementation used to deploy new cloned accounts.
     constructor(address erc4337) payable {
         implementation = erc4337;
     }
 
-    /// @dev Deploys an ERC4337 account and returns its deterministic address.
-    /// @param owners the initial set of addresses and or public keys that should be able to control the account
-    /// @param nonce the nonce of the account, allowing multiple accounts with the same set of initial owners to exist
+    /// @notice Deploys an ERC-4337 account and returns its deterministic address.
+    ///
+    /// @dev The account is deployed behind a minimal ERC1967 proxy whose implementation points to
+    ///      the registered ERC-4337 `implementation`.
+    /// @dev The `owners` parameter is a set of addresses and/or public keys depending on the signature
```

Comment:
I find this comment confusing, what does 1271 or webauthn have to do with it? 

I think we can just say "an array of initial owners of the CoinbaseSmartWallet"

Reply:
Well I was not sure between keeping things generic or giving the details. I picked the later approach as the contracts are already tightly coupled and if you provide "an array of initial owners of the CoinbaseSmartWallet" without taking care of what they are it might revert.